docs/src/spec/auth.md

```diff
@@ -63,26 +63,24 @@ pubkyauth:///
  ```
  and finally show that URL as a QR code to the user.
  
-4. The `Authenticator` app scans that QR code, parse the URL, and show a consent form for the user..
+4. The `Authenticator` app scans that QR code, parses the URL and shows a consent form for the user.
 5. The user decides whether or not to grant these capabilities to the `3rd Party App`.
-7. If the user approves, the `Authenticator` then uses their Keypair, to sign an [AuthToken](#authtoken), then encrypt that token with the `client_secret`, then calculate the `channel_id` by hashing that secret, and send that encrypted token to the callback url, which is the `relay` + `channel_id`.
-8. `HTTP Relay` forwards the encrypted AuthToken to the `3rd Party App` frontend.
-9. And confirms the delivery with the `Authenticator`
-10. `3rd Party App` decrypts the AuthToken using its `client_secret`, read the `pubky` in it, and send it to their `homeserver` to obtain a session.
+7. If the user approves then the `Authenticator` uses their Keypair to sign an [`AuthToken`](#authtoken-encoding), then encrypts that token with the `client_secret`. The `channel_id` is then calculated by hashing that secret and the encrypted token is sent to the callback url, which is the `relay` + `channel_id`.
```

Comment:
should be point 6 which is missing
I also think the first comma should remain instead of 'then'